lib/EnsEMBL/REST/Model/Phenotype.pm

```diff
@@ -127,4 +127,76 @@ sub fetch_features{
   return \@phenotype_features;
 }
 
+=head fetch_features_by_region
+
+extract phenotype features by region
+=cut
+
+sub fetch_features_by_region {
+
+  my $self      = shift;
+  my $species   = shift;
+  my $slice     = shift;
+
+  my $c = $self->context();
+
+  my $phenfeat_ad = $c->model('Registry')->get_adaptor($species,'variation', 'phenotypefeature');
+
+  my $pf_type = $c->request->parameters->{feature_type};
+  my $only_phenotypes = $c->request->parameters->{only_phenotypes};
+
+  my $pfs = $phenfeat_ad->fetch_all_by_Slice_with_ontology_accession($slice, $pf_type);
+
+  my %record_data;
+  my %phenotype_data;
+
+  foreach my $pf(@{$pfs}){
+
+    my $object_id = $pf->object_id; 
+    my $phe_id    = $pf->{_phenotype_id};
```

Comment:
That's a quicker way to get the internal phenotype_id. I should write a method in the Variation API to get it more elegantly